### PR TITLE
feat(variant_mapper): update msrp on variants

### PR DESCRIPTION
**Pull request by Jhalter5Stones**
update msrp as default price on variants

### CODE REVIEW

**Comment by Jhalter5Stones** [comment]
@mrogers-5s Ready for review. Let me know if I am missing something here.

https://trello.com/c/gvwCTFES/240-how-can-we-use-msrp-and-customer-groups

File: lib/mapper/variant_mapper.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def self.map(variant, option_values, product_id, variant_id=nil, not_purchasable
                     price: variant['offers'] && variant['offers'][0] ? variant['offers'][0]['price'] : '0',
                     cost_price: nil,
                     sale_price: nil,
-                    retail_price: nil,
+                    retail_price: variant['offers'] && variant['offers'][0] ? variant['offers'][0]['price'] : '0',
```
Comment (mrogers-5s) [inline lib/mapper/variant_mapper.rb:12]: I think we probably want to leave it as `nil` if it's not provided.

Comment (Jhalter5Stones) [reply lib/mapper/variant_mapper.rb]: @mrogers-5s So like this?

**Review by mrogers-5s — APPROVED** [review]
LGTM -- squash those commits and send it for final review

**Comment by Jhalter5Stones** [comment]
@jspizziri Can you final review this?

**Pull request merged**